package.json

```diff
@@ -20,7 +20,10 @@
     "bootstrap",
     "underscores"
   ],
-  "dependencies": {},
+  "dependencies": {
```

Comment:
These npm packages are unneeded. You forgot to remove these after trying to figure out how to use `wget` the other day. :) Reset this back to to empty braces.